lisa/tools/lsblk.py

```diff
@@ -1,10 +1,51 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+import re
+from dataclasses import dataclass
+from typing import List
+
 from lisa.executable import Tool
+from lisa.util import find_patterns_in_lines
+
+
+@dataclass
+class PartitionInfo(Tool):
```

Comment:
Use the class from df.

Reply:
They both contain different kind of information. Is your intent here to merge both these classes into one class and fill only the required entries in the class depending on the command?

Reply:
How about we move ` get_partition_information(self)` to disk features and populate info from both lsblk and df?

Reply:
see comments in `df`. It should be the same PartitionInformation class, and the disk/partition are not same level things. Let's keep partition is out of disks for now.